lld/test/ELF/lto/cache.ll

```diff
@@ -53,10 +53,10 @@
 
 ; RUN: rm -fr cache && mkdir cache
 ; RUN: ld.lld --thinlto-cache-dir=cache --save-temps -o out b.bc a.bc -M | FileCheck %s --check-prefix=MAP
-; RUN: ls out1.lto.o a.bc.0.preopt.bc b.bc.0.preopt.bc
+; RUN: ls out.lto.a.o a.bc.0.preopt.bc b.bc.0.preopt.bc
 
-; MAP: llvmcache-{{.*}}:(.text)
-; MAP: llvmcache-{{.*}}:(.text)
+; MAP: out.lto.b.o:(.text)
```

Comment:
Confused about what changed in the latest commit to cause these lines to need updating - was this just a missed test update in the prior commit?

Reply:
This is a new test. I noticed the `-Map` difference, so I pre-committed a test and changed this PR to demonstrate the difference.